### PR TITLE
Making the escaping of arguments optional and removing hardcoded vari…

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
…able

Making CommandLine more flexible.

### CODE REVIEW

File: tests/integration/features/bootstrap/CommandLine.php [hunk 1]
```diff
@@ -29,19 +29,18 @@
 	/** @var string stderr of last command */
 	private $lastStdErr;
 
-	/** @var string */
-	protected $ocPath = '../..';
```
Comment (PVince81) [inline tests/integration/features/bootstrap/CommandLine.php]: where did it go ? part of another PR ?

Comment (SergioBertolinSG) [inline tests/integration/features/bootstrap/CommandLine.php]: It was already here https://github.com/owncloud/core/blob/master/tests/integration/config/behat.yml#L25

Comment (PVince81) [inline tests/integration/features/bootstrap/CommandLine.php]: ah ok, curious to see if tests will still pass

**Comment by PVince81** [comment]
:+1:

**Pull request merged**